internals/overlord/servstate/manager_test.go

```diff
@@ -420,24 +453,31 @@ func (s *S) TestStartBadCommand(c *C) {
 }
 
 func (s *S) TestCurrentUserGroup(c *C) {
-	s.setupEmptyServiceManager(c)
+	s.newServiceManager(c)
+	s.planAddLayer(c, testPlanLayer)
 
 	current, err := user.Current()
 	c.Assert(err, IsNil)
 	group, err := user.LookupGroupId(current.Gid)
 	c.Assert(err, IsNil)
 
 	outputPath := filepath.Join(c.MkDir(), "output")
-	layer := parseLayer(c, 0, "layer", fmt.Sprintf(`
+	layer := `
 services:
     usrtest:
         override: replace
-        command: /bin/sh -c "id -n -u >%s; sync; %s; sleep %g"
+        command: /bin/sh -c "id -n -u >%s; {{.NotifyDoneCheck}}; sleep %g"
         user: %s
         group: %s
-`, outputPath, s.insertDoneCheck(c, "usrtest"), shortOkayDelay.Seconds()+0.01, current.Username, group.Name))
-	err = s.manager.AppendLayer(layer)
-	c.Assert(err, IsNil)
+`
+	s.planAddLayer(c, fmt.Sprintf(
+		layer,
+		outputPath,
+		shortOkayDelay.Seconds()+0.01,
+		current.Username,
+		group.Name,
+	))
```

Comment:
Thanks for these code formatting cleanups -- they do read more clearly now.

Reply:
Pleasure. Resolving.